src/pages/office/components/d255/d255.html

```diff
@@ -1,7 +1,7 @@
 <ion-row *ngIf="display" class="mes-validated-row mes-row-separator" id="d255-card" [formGroup]="formGroup">
   <div class="validation-bar" [class.ng-invalid]="invalid">
   </div>
-  <ion-col class="component-label" col-32 align-self-center>
+  <ion-col class="component-label" col-36 align-self-center>
```

Comment:
These components need moving to the pass finalisation components folder as the end result is to remove them from the office.